### DISCUSSION TITLE
DragonFlight expansion is not active being version 10.0.2

### DISCUSSION THREAD

**Discussion by jhoancito** [TrinityCore/TrinityCore#28683] [General]
DragonFlight expansion is not active being version 10.0.2 how is that expansion activated since which server is the dragonflight expansion or is it not yet implemented at the source

**Comment by mdX7** [comment]
Set `Expansion` in `worldserver.conf` to 9 and make sure `expansion` field in `account` table in `auth` is also set to 9 for your account. Restart server after changes preferably.
